### PR TITLE
Refresh branch: faq folder files converted from .rst to .md

### PR DESCRIPTION
**Pull request by kentydotcom**
Converted all faq rst files into md (English).
Checked via local docker Jekyll renderer.

### CODE REVIEW

File: faq/client.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+<!--
```
Comment (vaporos) [inline faq/client.md]: Copyright should come after the first header and look like line one here (with the appropriate copyright):

https://raw.githubusercontent.com/Cargill/splinter-docs/master/docs/0.4/index.md

(same applies to all other headers that look like this)

File: faq/client.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  \# Copyright (c) 2018, Intel Corporation.
+    \# © Copyright 2020, Dr Kent G LAU, <kenty@kenty.com>.
+  \# Licensed under Creative Commons Attribution 4.0 International License
+  \# <https://creativecommons.org/licenses/by/4.0/>
+  \-\
+  -->
+
+# Sawtooth FAQ: Client
+  
+---
```
Comment (vaporos) [inline faq/client.md]: These division lines should be omitted in the markdown. (same for other instances)

File: faq/client.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+<!--::: {.mininav}
```
Comment (vaporos) [inline faq/client.md]: Remove the commented-out pieces like this that don't get rendered.

File: faq/faq.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Answers to frequently asked questions for Hyperledger Sawtooth. Many
+questions and answers originated from the \#sawtooth Rocket.Chat channel
+at <https://chat.hyperledger.org/channel/sawtooth>
+
+---
+
+## Sawtooth FAQ Sections
+
+-   [Hyperledger Sawtooth in General](sawtooth)
```
Comment (vaporos) [inline faq/faq.md]: These links are broken. You can test the site by running "just run" (after installing 'just') and pointing the browser to http://localhost:4000/.

The correct way to write these particular lines is:

```
 -   [Hyperledger Sawtooth in General]({% link faq/sawtooth.md %})
```

File: faq/faq.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+(`git commit -s`). This sign-off means you agree the commit satisfies
+the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).\]
+
+<!--
+::: {.mininav}
+[NEXT](/faq/sawtooth/)
+:::
+-->
+
+© Copyright 2018, Intel Corporation.
```
Comment (vaporos) [inline faq/faq.md]: Remove all copyrights at the end of the files (this should now be in the comment on the top of the files).

**Review by vaporos — CHANGES_REQUESTED** [review]

**Comment by vaporos** [comment]
Please augment the commit messages following these suggestions as a guide - https://chris.beams.io/posts/git-commit/

**Comment by vaporos** [comment]
Please move the internationalization to a separate PR. For reference, here is a possible plugin we can use to help with this piece in jekyll - https://github.com/kurtsson/jekyll-multiple-languages-plugin.

**Comment by arsulegai** [comment]
@rbuysse @rberg2 The PR build fails because @kentydotcom is not part of approved list.

**Comment by rbuysse** [comment]
> @rbuysse @rberg2 The PR build fails because @kentydotcom is not part of approved list.

👍 Kenty's been added to the authorized builders list.

**Review by vaporos — DISMISSED** [review]

**Comment by vaporos** [comment]
That should have read "There is a comment".

**Comment by agunde406** [comment]
This work has been completed and merged into main.

**Pull request closed without merging**